### ISSUE TITLE
On-disk support (lustre)

### ISSUE THREAD

**Issue by danthegoodman1** [danthegoodman1/icedb#117]
Have a mode where it operates off disk instead of S3 in case of using something like AWS fsx lustre. That might provide better managed performance, especially for reading the log.

**Comment by loicalleyne** [comment]
Is this for both the metastore and the parquet data files? I suppose this would work with a local RAID drive setup as well?

**Comment by danthegoodman1** [comment]
Yeah it would be really for any locally mounted disk most likely, and yes both would be supported as they are currently colocated under one parent directory for the table

**Comment by danthegoodman1** [comment]
Also useful for JuiceFS

**Comment by danthegoodman1** [comment]
Should be able to chose location for log and data separately, so could chose to have log on disk but data in s3
